p2p/src/net/default_backend/backend.rs

```diff
@@ -464,7 +464,26 @@ where
             PeerEvent::MessageReceived { message } => self.handle_message(peer_id, message),
 
             PeerEvent::ConnectionClosed => {
-                self.pending.remove(&peer_id);
+                if let Some(pending_peer) = self.pending.remove(&peer_id) {
+                    match pending_peer.peer_role {
+                        PeerRole::Inbound => {
+                            // Just log the error
+                            log::debug!("inbound pending connection closed unexpectedly");
```

Comment:
This probably should be warn, not debug.

Reply:
Sure, updated.